hazelcast/src/main/java/com/hazelcast/core/EntryView.java

```diff
@@ -40,23 +40,17 @@
 
     /**
      * Returns the cost (in bytes) of the entry.
-     * <p/>
-     * <p><b>Warning:</b></p>
      * <p>
-     * This method returns -1 if statistics is not enabled.
-     * </p>
+     * <b>Warning:</b> This method returns -1 if statistics are not enabled.
```

Comment:
This class is now used for more structures? Should we say that the method can return -1 also if the statistics are not supported/implemented?

Reply:
Well spotted. This is a design decision. `EntryView` was used by `IMap` and `ReplicatedMap` (via `ReplicatedMapEntryView implements EntryView`). `ICache` uses it's own base class (`CacheEntryView extends EvictableEntryView`), which has the same information, just one field is named differently (`getHits()` vs. `getAccessHit()`).

For the split-brain healing I wanted a common interface for all data structures. Since I found no hard coupling between `com.hazelcast.core.EntryView` and the map data structures, I thought it would be good to re-use it.

If we keep this inheritance, it makes sense to adjust the JavaDoc (did this already, since the `EntryView` mentioned "map" somewhere).

Reply:
I've fixed the JavaDoc and also added the `-1` to the according fields of `SimpleSplitBrainEntryView` implementation in `SplitBrainEntryViews`.